src/main/java/Example2.java

```diff
@@ -42,5 +47,3 @@ void bar() {
     }
   }
 }
-
-class Example2a {}
```

Comment:
Is there a reason you are disallowing multiple top-level classes?  I don't think I've come across an example of it in the iq codebase, but I didn't think it was something we were prohibiting.

Reply:
I don't like them?  We could easily allow this, that was just one of the default rules.  I don't really have strong feelings about what the rules are, just that they are applied consistently.